examples/cartesian_path.py

```diff
@@ -78,13 +81,34 @@
 
 plt.ion()
 plt.figure()
-plt.title("Joint Position Trajectories")
+plt.title("Pyroboplan - Joint Position Trajectories")
+
+if TOPP_RA:
+    import toppra as ta
+
+    # replan path with TOPP-RA
+    vlims = model.velocityLimit
+    alims = 10 * np.ones_like(vlims)
```

Comment:
I think this was just an example I used temporarily, but we probably need to add comments here.

Namely that URDF does not support acceleration limits, so you have to define this information separately and we're just using some provisional values.

At some point, I intend to add a "wrapper" around the Pinocchio model where acceleration/jerk limits can be specified, so a temporary comment like the above is OK for now.

Reply:
Will such a comment be ok?
```suggestion
    # URDF does not support acceleration limits, you must specify this value yourself or use this temporary one
    alims = 10 * np.ones_like(vlims)
```

Reply:
That works for now, thanks!